drivers/virtio/Kconfig

```diff
@@ -0,0 +1,45 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig DRIVERS_VIRTIO
+	bool "Virtio Device Support"
+	depends on !OPENAMP
+	default n
+	---help---
+
+if DRIVERS_VIRTIO
+menuconfig DRIVERS_VIRTIO_MMIO_NUM
+	int "The number of virtio mmio devices"
+	default 1
+	---help---
+
+menuconfig DRIVERS_VIRTIO_MMIO_BASE
+	hex "Virtio-mmio base address"
+	---help---
+
+menuconfig DRIVERS_VIRTIO_MMIO_REGSIZE
+	hex "Virtio-mmio register size"
```

Comment:
could add the default value 0 since it is suitable for single device

Reply:
done